### DISCUSSION TITLE
Would it be possible to implement lazy loading per page?

### DISCUSSION THREAD

**Discussion by warben0** [jgraph/drawio#4132] [General]
* [ X ] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [ X ] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
I would like for example when you open your diagram as viewer, that it only loads the page you enter, then loads the other pages as you enter those pages, this would decrease loading times for bigger diagrams

**Describe alternatives you've considered**
Splitting diagrams, but this is not ideal as a single page is easier for users to follow

**Additional context**
I've seen loading times increase quite a bit as my diagram grows, just wondering if this is possible to implement?

**Comment by davidjgraph** [comment]
This isn't possible under the current architecture.

**Comment by warben0** [comment]
Where is the limitation may i ask?

**Reply by davidjgraph** [reply]
It's one file.
